### PR TITLE
win_virtio_driver_install_from_update:add fwcfg test

### PR DESCRIPTION
**Pull request by leidwang**
ID:2142456
Signed-off-by: Leidong Wang <leidwang@redhat.com>

### CODE REVIEW

**Comment by leidwang** [comment]
JOB LOG    : /root/avocado/job-results/job-2022-11-14T00.31-5d7d658/job.log
 (1/1) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.win_virtio_driver_install_from_update.with_fwcfg.q35: STARTED
 (1/1) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.win_virtio_driver_install_from_update.with_fwcfg.q35: PASS (302.04 s)

**Comment by leidwang** [comment]
Hi @xiagao @vivianQizhu Could you help to review this patch?Thanks!

**Review by vivianQizhu — APPROVED** [review]

File: qemu/tests/cfg/win_virtio_driver_install_from_update.cfg [hunk 1]
```diff
@@ -126,3 +126,8 @@
             device_name = "VirtIO FS Device"
             device_hwid = '"PCI\VEN_1AF4&DEV_105A"'
             fs_binary_extra_options = " -o cache=auto"
+        - with_fwcfg:
+            vmcoreinfo = yes
```
Comment (xiagao) [inline qemu/tests/cfg/win_virtio_driver_install_from_update.cfg:130]: @leidwang Could you limit the guests and hosts that fwcfg support?

Comment (xiagao) [reply qemu/tests/cfg/win_virtio_driver_install_from_update.cfg:130]: There is a limitation on the top, it's okay with me.   
 only Windows
    no WinXP WinVista Win7 Win8 Win8.1 Win2003
    no Win2008 Win2008..r2 Win2012 Win2012..r2

**Review by xiagao — APPROVED** [review]
ack.

**Pull request merged**